libs/blocks/global-navigation/blocks/profile/profile.css

```diff
@@ -0,0 +1,133 @@
+:root {
+  --border-color: #e1e1e1;
```

Comment:
I think this may be a bit generic and may interfere with other components and their variables in an unexpected way. Since I'm assuming this is used just in the context of the profile, why not attach it to an actual profile element, the parent of all profile elements? Either that or make the variable name a bit more specific to FEDS/profile.